pom.xml

```diff
@@ -758,6 +760,19 @@
                 <artifactId>aliyun-sdk-datahub</artifactId>
                 <version>${datahub.version}</version>
             </dependency>
+
+            <dependency>
```

Comment:
Hi, please move connector dependency to connector's pom.xml. Reference: https://github.com/apache/incubator-seatunnel/pull/2630

Reply:
Ok, they have been moved to connector's pom.xml